test_common/page_objects/form_submission_page.rb

```diff
@@ -2,6 +2,17 @@ module ET3
   module Test
     class FormSubmissionPage < BasePage
       set_url '/respond/form_submission'
+
+      element :submission_confirmation, :css, '.submission-confirmation'
+
+      element :reference_number, :css, '.reference-number'
+
+      element :download_pdf, :css, '.download-pdf'
+
+      element :govuk_button, :css, 'a.button.button-start'
+      def finish
+        govuk_button.click
```

Comment:
Funny name for a button ?  Can this not be more like what the user sees ?

Reply:
Good point, fixed in fc9815a